modules/schematics/src/effect/index.spec.ts

```diff
@@ -304,9 +303,31 @@ describe('Effect Schematic', () => {
       `${projectPath}/src/app/effects/foo/foo.effects.ts`
     );
 
-    expect(content).toMatch(
-      /import \{ FooActionTypes, FooActions } from '\.\.\/\.\.\/actions\/foo\/foo\.actions';/
-    );
+    expect(content).toMatchInlineSnapshot(`
```

Comment:
`toMatchInlineSnapshot` is used here because it was easier to see what was going on (and wrong).